librz/il/il_export.c

```diff
@@ -335,6 +404,257 @@ static void il_opdmp_append(RzILOpPure *op, RzStrBuf *sb, PJ *pj) {
 	il_op_param_2("append", op->op.append, pure, high, pure, low);
 }
 
+static void il_opdmp_float(RzILOpPure *op, RzStrBuf *sb, PJ *pj) {
+	RzILOpArgsFloat *opx = &op->op.float_;
+	if (sb) {
+		rz_strbuf_appendf(sb, "(float %d ", opx->r);
+		il_op_pure_resolve(opx->bv, sb, pj);
+		rz_strbuf_append(sb, " )");
```

Comment:
Could be simplified by using `return` and removing `else`

Reply:
@XVilka when reviewing this file, other il_opdmp_* functions also have `if ... else` style, Does il_opdmp_* for float should keep the same style with them ?

Reply:
@Heersin it's ok then, just a suggestion. You can keep it as is.